examples/shipments/lowest_rate/lowest_rate.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/EasyPost/easypost-go"
+)
+
+func main() {
+	apiKey := os.Getenv("EASYPOST_API_KEY")
+	if apiKey == "" {
+		fmt.Fprintln(os.Stderr, "missing API key")
+		os.Exit(1)
+		return
+	}
+	client := easypost.New(apiKey)
+
+	// Create and verify addresses
+	toAddress, err := client.CreateAddress(
```

Comment:
Like the unit test, let's make the example use raw data passed into the fields of the Shipment object instead of making a half-dozen API calls.